### DISCUSSION TITLE
Published results for MTSC on UEA's archive - where are they?

### DISCUSSION THREAD

**Discussion by binhminh98** [alan-turing-institute/sktime#3186] [Q&A] [answered]
Hi guys, I’m doing my dissertation project on sktime classification module. I have a question: Where can I find the published results for MTSC on UEA's archive?

I can't find it on the website: http://www.timeseriesclassification.com/results.php nor the link: http://www.timeseriesclassification.com/mtsc_bakeoff.php

**Comment by haskarb** [comment]
Are you not able to [download](http://www.timeseriesclassification.com/results/AllAccuracies.zip) zip file?

**Reply by binhminh98** [reply]
Hi, I think the zip files are for univariate time series only. I am finding the published results for this paper: https://link.springer.com/article/10.1007/s10618-020-00727-3

**Reply by haskarb** [reply]
Ok, I'm not sure about it then. If this helps, I have some results on [this](https://ieeexplore.ieee.org/document/9346402)

**Reply by binhminh98** [reply]
Thank you very much, I will have a look at it

**Reply by fkiraly** [reply]
@haskarb, if you have code, would you like to help building `sktime`'s benchmarking functionality? One of our goals is to be able to run classifier benchmark experiments without any reliance on the UEA infrastructure

**Comment by fkiraly** [comment]
FYI @MatthewMiddlehurst, @TonyBagnall

**Comment by TonyBagnall** [comment, marked as answer]
hi, @binhminh98  all the results are here https://timeseriesclassification.com/

**Reply by fkiraly** [reply]
could you maybe be a bit more specific, i.e., which experiment or which archive it is precisely in?

**Reply by fkiraly** [reply]
@TonyBagnall, I unmarked your results as answer, since it is clearly unhelpful. @binhminh98, in his original post, already mentioned the very location you provide and said he cannot find it there.

Providing the link again which @binhminh98 already provided adds no information except asserting that they have not looked properly, which is also not very welcoming.

If you provide an answer, please explain to @binhminh98 where specifically on the webpage they can find what they are looking for.

**Reply by fkiraly** [reply]
@binhminh98, I would like to reassure you: it is not clear where the results are or whether they are there, e.g., I couldn't tell and cannot easily infer from the webpage where they would be. So, your question is very reasonable.

**Reply by MatthewMiddlehurst** [reply]
The link in the original post and the one given are different.

The results have now been uploaded to that page under the header "TSC/TSCL Results".

**Reply by MatthewMiddlehurst** [reply]
"Download the tsml classification accuracy results for the 26 UEA multivariate TSC problems presented in the multivariate bake off and the HC2 paper."

**Reply by fkiraly** [reply]
I see, so when @TonyBagnall answered, the results haven't actually been there?

I'd recommend we wait for @binhminh98 to confirm that what is there is indeed what he was looking for before we mark this as solved.

**Reply by MatthewMiddlehurst** [reply]
They were uploaded, the bakeoff page linked in the original discussion is still being updated.

I sent @binhminh98 the results soon after the original post, but we can wait to see if they reply.

**Reply by fkiraly** [reply]
@MatthewMiddlehurst, do I interpret you statement correctly as:

* you assert the results will be uploaded to the linked page
* you don't want to be more precise as to where they will be
* instead you preferred to send them to @binhminh98 by direct message
* you assert @binhminh98 has received this and thought the information to be satisfactory

Is this correct?

**Reply by MatthewMiddlehurst** [reply]
The results are already uploaded to the page Tony linked and have been for over a week at this point, but will also be uploaded to the supporting page for the paper. I did point out where they are. 

I am unsure why this requires an interrogation.

**Reply by TonyBagnall** [reply]
I dont see why this conversation has become an interrogation of @MatthewMiddlehurst. Please keep things polite and in line with our code of conduct. @binhminh98 has finished his MSc and may not reply. Please note that timeseriesclassification.com is part of UEA infrastructure, and this whole topic would be better raised as an issue here https://github.com/uea-machine-learning/tsml_repo

**Reply by binhminh98** [reply]
Hi @all,

I have found what I was looking for and did not get any notifications from this thread since then. The website is very clear now on these MTSC results and I will mark this discussion as answered. Thank you very much.
